project/Dependencies.scala

```diff
@@ -256,6 +257,7 @@ object Dependencies {
         jacksonJsr310,
         jacksonParameterNames,
         jacksonCbor,
+        lz4Java,
```

Comment:
Would have been nicer if this wasn't a mandatory dependency if not using it though, is it possible to mark it as "provided"?